apps/web/drizzle/0000_graceful_spectrum.sql

```diff
@@ -0,0 +1,46 @@
+CREATE SCHEMA "latitude";
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."commits" (
+	"serial" serial PRIMARY KEY NOT NULL,
+	"next_commit_id" integer,
+	"title" varchar(256) NOT NULL,
+	"description" text,
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"updated_at" timestamp DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."prompt_versions" (
+	"serial" serial PRIMARY KEY NOT NULL,
+	"prompt_id" integer NOT NULL,
+	"path" varchar NOT NULL,
+	"content" text NOT NULL,
+	"hash" varchar NOT NULL,
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"updated_at" timestamp DEFAULT now() NOT NULL,
+	CONSTRAINT "prompt_versions_path_unique" UNIQUE("path")
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."prompts" (
+	"serial" serial PRIMARY KEY NOT NULL,
+	"path" varchar NOT NULL,
+	"content" text NOT NULL,
+	"hash" varchar NOT NULL,
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"updated_at" timestamp DEFAULT now() NOT NULL,
+	CONSTRAINT "prompts_path_unique" UNIQUE("path")
+);
```

Comment:
Missing `prompt_id`, which should be UNIQUE here too

Reply:
But also have `prompt_id` pointing to a prompt?